packages/uiweb/src/lib/components/chat/ChatPreviewList/ChatPreviewList.tsx

```diff
@@ -546,7 +546,7 @@ export const ChatPreviewList: React.FC<IChatPreviewListProps> = (
       chatRequestStream.constructor === Object
     ) {
       console.debug('Chat request stream', chatRequestStream);
-      if (options.listType === CONSTANTS.CHAT.LIST_TYPE.REQUESTS) {
+      if (options.listType === CONSTANTS.CHAT.LIST_TYPE.CHATS) {
```

Comment:
this is fine for sender but wont this change cause issue for the receiver, instead of the chat apearing in the REQUESTS side it will come in the CHATS side?

Reply:
@mishramonalisha76 fixed it.